### PR TITLE
#128: Merged word & steps

### PR DESCRIPTION
**Pull request by afijal**

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -427,14 +433,14 @@ function ProblemIsUnchanged(buttonElement) {
 // CREATE NEW HISTORY ROW FROM CURRENT CONTENT
 // @param {mathContent} latex for new active area after being cleaned.
 // @return {nothing} No return value
-function NewMathEditorRow(mathContent) {
+function NewMathEditorRow(mathContent, showStepLabel, cleanup) {
```
Comment (oskarhinc) [inline scripts/scripts.js]: Do we need two flags here? Aren't they always opposites?

File: scripts/scripts.js [hunk 2]
```diff
@@ -126,13 +126,19 @@ function OrdinalSuffix(i) {
...
-function HTMLForRow(stepNumber, math, annotation, showTrash) {
+function HTMLForRow(stepNumber, math, annotation, showTrash, showStepLabel) {
 	let html = '<div class="row mathStep" data-step="'+stepNumber+'" data-equation="'+math+'" data-annotation="'+annotation+'">';
-	html += '<div class="col-md-6">';
+	html += '<div class="col-md-1">';
 	html +=   '<span role="heading" aria-level="3">';
-	html +=     '<span class="SROnly">' + OrdinalSuffix(stepNumber) +' step</span>';
-	html +=     '<span class="stepHeader" aria-hidden="true">Step '+stepNumber+':</span>';
+	if (showStepLabel) {
+		html +=     '<span class="SROnly">' + OrdinalSuffix(stepNumber) +' step</span>';
```
Comment (oskarhinc) [inline scripts/scripts.js]: I believe white spaces after '+=' were used to display HTML tags in code with an indentation, but as you used 'if' statement, it's no longer needed.

File: scripts/scripts.js (hunk 1 above)
Comment (afijal) [reply scripts/scripts.js]: Right

File: scripts/scripts.js (hunk 2 above)
Comment (afijal) [reply scripts/scripts.js]: Good catch!

**Comment by afijal** [comment]
I've committed corrections

**Pull request merged**